CMakeLists.txt

```diff
@@ -84,8 +92,21 @@ macro(build_libyaml)
   set(yaml_LIBRARIES yaml)
 endmacro()
 
-build_libyaml()
-set(extra_test_dependencies libyaml-0.2.5)
+# Skip building yaml if the expected version is already present in the system
+if(yaml_FOUND)
+  if("${yaml_VERSION}" VERSION_EQUAL 0.2.5)
```

Comment:
Maybe `VERSION_GREATER_EQUAL` would be better - that's what we're doing in other vendor packages, I think.

Reply:
Yes other vendor packages do that.
However it was pointed out that here we need exact version.

Reply:
IIUC version pegging is what enables us to claim this is a QL1 package. API/ABI compatibility is not guaranteed by `libyaml`.